lib/lpc55-hashcrypt/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+/// proc macro to generate iocon function for LPC55
+extern crate proc_macro;
+use proc_macro::TokenStream;
+extern crate proc_macro2;
+use quote::quote;
+
+#[proc_macro]
```

Comment:
Can we delete this code if we're not using it?

Reply:
This code has been previously removed.